src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -156,6 +156,27 @@
   @Setter(AccessLevel.PROTECTED)
   protected transient ComputeEngineCloud cloud;
 
+  public static List<Metadata.Items> mergeMetadataItems(
```

Comment:
Looks like this is only used within this class, makes sense to restrict the scope to private, unless you want to make package scoped for unit testing purposes.